app/Console/Commands/WaitingLists/CheckMembersMeetActivityRules.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace App\Console\Commands\WaitingLists;
+
+use App\Console\Commands\Command;
+use App\Models\Mship\Account;
+use App\Models\NetworkData\Atc;
+use App\Models\Training\WaitingList;
+use App\Notifications\Training\WaitingListRemovalAdded;
+use App\Notifications\Training\WaitingListRemovalCancelled;
+use App\Notifications\Training\WaitingListRemovalCompleted;
+use App\Notifications\Training\WaitingListRemovalReminder;
+use Carbon\Carbon;
+
+class CheckMembersMeetActivityRules extends Command
+{
+
+    protected $minutesRequired = 720; // 12 hours is represented as 720 minutes
+
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'waitinglists:hourchecker';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Checks ATC hour totals for members who are on an ATC waiting list and marks inactive members for automatic removal from the list.';
+
+    /**
+     * A local cache of account activity to prevent duplicate calculations of a members hours
+     * 
+     * @var array
+     */
+    protected $accountActivityCache = [];
+
+    /**
+     * Executes all necessary console commands.
+     */
+    public function handle()
+    {
+        foreach($this->retrieveHourEnforcedWaitingLists() as $waitingList){
+            $this->identifyNewNonEligibleWaitingListAccounts($waitingList);
+            $this->processWaitingListAccountsMarkedForRemoval($waitingList);
+       }
+       
+    }
+
+    /**
+     * Retrieves training waiting lists which enforce hour requirements
+     */
+    protected function retrieveHourEnforcedWaitingLists()
```

Comment:
Every ATC waiting list, by virtue of being that type, has an hour check built in by default (see : https://github.com/VATSIM-UK/core/blob/c3ac806defa080dd5a67be737aae3c24788148e1/app/Models/Training/WaitingList/WaitingListAccount.php#L138)

I'd argue that actually this column is not required on this basis and that you should just filter for ATC list types.

Reply:
Currently it is the case that all ATC waiting lists require the hour check, however, when discussing Adam's plans with him to see what I could assist with, he expressed interest in migrating the OBS-S1 waiting list from CTS to Core. 

In that event, the waiting list would not require an hour check, so the hard coded solution currently in place would need changing. 

Since I was working heavily on the hour check aspect of the waiting list in this task, it made sense to add this as a waiting list specific setting now.

Reply:
As you say, many more changes are required to have an ATC list that doesn't enforce any hour requirements, so I'd suggest keeping the status quo until the need arises.